ViewPumpSample/src/main/java/io/github/inflationx/viewpump/sample/MainActivity.java

```diff
@@ -0,0 +1,68 @@
+package io.github.inflationx.viewpump.sample;
+
+import android.content.Context;
+import android.os.Bundle;
+import android.os.Handler;
+import android.os.Looper;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+
+import io.github.inflationx.viewpump.ViewPumpContextWrapper;
+
+import static butterknife.ButterKnife.findById;
+
+public class MainActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        final Toolbar toolbar = findById(this, R.id.toolbar);
+        setSupportActionBar(toolbar);
+
+        // Inject pragmatically
+        getSupportFragmentManager()
+                .beginTransaction()
+                .replace(R.id.container, PlaceholderFragment.getInstance())
+                .commit();
+
+
+        final Handler handler = new Handler(Looper.getMainLooper());
```

Comment:
I think we can remove all this from the demo stuff and keep it limited to simple ViewPump examples